app/router.js

```diff
@@ -26,7 +26,8 @@ Router.map(function() {
   this.route('security');
   this.route('sponsors');
   this.route('team');
-  this.route('tomster', function() {
+  this.route('tomster', { path: '/*mascots' });
```

Comment:
This line should be removed as it is insufficient for redirecting, which will be taken care of by https://github.com/ember-learn/ember-website/pull/133.